main.go

```diff
@@ -182,7 +184,7 @@ func main() {
 func webhooks() []webhookwrapper.WebhookRegistrator {
 	return []webhookwrapper.WebhookRegistrator{
 		{
-			Registrator: (&v1alpha2.PolicyServer{}).SetupWebhookWithManager,
+			Registrator: (&policiesv1.PolicyServer{}).SetupWebhookWithManager,
```

Comment:
shouldn't we still be looking after v1alpha2 policies?

Reply:
this seems to be the same as https://github.com/kubewarden/kubewarden-controller/pull/247#discussion_r890870189